cmd/minishift/cmd/start.go

```diff
@@ -87,6 +82,13 @@ var minishiftToClusterUp = map[string]string{
 
 // init configures the command line options of this command
 func init() {
+	startCmd = &cobra.Command{
```

Comment:
I think below error shows that in getting to initialization loop which has cyclic dependency 
```
$ make
go install -pkgdir=/home/lmohanty/GoCode/src/github.com/minishift/minishift/out/bindata -ldflags="-X github.com/minishift/minishift/pkg/version.version=1.0.1 -X github.com/minishift/minishift/pkg/version.isoVersion=v1.0.2 -X github.com/minishift/minishift/pkg/version.openshiftVersion=v1.5.1" ./cmd/minishift
# github.com/minishift/minishift/cmd/minishift/cmd
cmd/minishift/cmd/start.go:70: initialization loop:
	/home/lmohanty/GoCode/src/github.com/minishift/minishift/cmd/minishift/cmd/start.go:70 startCmd refers to
	/home/lmohanty/GoCode/src/github.com/minishift/minishift/cmd/minishift/cmd/start.go:104 runStart refers to
	/home/lmohanty/GoCode/src/github.com/minishift/minishift/cmd/minishift/cmd/start.go:384 validateOpenshiftVersion refers to
	/home/lmohanty/GoCode/src/github.com/minishift/minishift/cmd/minishift/cmd/start.go:70 startCmd
Makefile:56: recipe for target '/home/lmohanty/GoCode/bin/minishift' failed
make: *** [/home/lmohanty/GoCode/bin/minishift] Error 2
```